src/azure-cli/azure/cli/command_modules/security/custom.py

```diff
@@ -253,22 +253,15 @@ def list_security_pricings(client):
     return client.list()
 
 
-def get_security_pricing(client, resource_name, resource_group_name=None):
-
-    if resource_group_name:
-        return client.get_resource_group_pricing(resource_group_name, resource_name)
+def get_security_pricing(client, resource_name):
 
     return client.get(resource_name)
 
 
-def create_security_pricing(client, resource_name, tier, resource_group_name=None):
-
-    if resource_group_name:
```

Comment:
Will this deletion likely to break user?

Reply:
AFAIk this option isnt available in the new api, im not sure if this is auto-generated or not. 
I was unable to reach this code using CLI . tried running commands 
az security pricing create with parameter "resourceGroup" but received error - unknown arguemnt
Is there a way to trace back how this is called and what the parameter name for this is?

![image](https://user-images.githubusercontent.com/22338979/98232266-1e469700-1f66-11eb-8ffa-4a634249f8ad.png)

Reply:
you could type --debug to get more verbose output to help.  If you want to see what options currently  there it is, you could run it with -h.  
If some IDEs you are using for debugging ,  it depends.

Reply:
Thanks. 

I might be mistaken, but its possible that this command is already broken. 
When running a command (in master, unchanged branch) with the resource group parameter:

_az security pricing show -n "virtualMachines" -g "resourceGroupName"_

[as per doc: ](https://docs.microsoft.com/en-us/cli/azure/security/pricing?view=azure-cli-latest#az_security_pricing_create-optional-parameters)

I get the following error which corresponds to the method being removed:

"CLIInternalError: The command failed with an unexpected error. Here is the traceback:
'PricingsOperations' object has no attribute 'get_resource_group_pricing'"

Reply:
I am okay if you are confirmed about it.